### ISSUE TITLE
Apollo 3 beta 41 broke apollo integration

### ISSUE THREAD

**Issue by pix2D** [jhen0409/react-native-debugger#478]
React Native Debugger app version: 0.10.7
React Native version: 0.61.5
Platform: iOS
Is real device of platform: No
Operating System: macOS

Apollo tab stopped working since the last beta update (`@apollo/client 3.0.0-beta.41`). I believe it _may_ be [this PR](https://github.com/apollographql/apollo-client/pull/6046) but not 100% sure.

```
Uncaught TypeError: Cannot read property 'getStore' of undefined
    at te.<anonymous> (RNDebuggerWorker.js:2)
    at te.s.emit (RNDebuggerWorker.js:2)
    at RNDebuggerWorker.js:2
    at RNDebuggerWorker.js:2
```

**Comment by darkbasic** [comment]
I confirm the issue, but did you check if it still works in Chrome with a non-native react app?

**Comment by KennyHammerlund** [comment]
to my knowledge, they have not released apollo devtools that is compatible with 3.0. This branch appears to fix it but it has not been merged yet.  
https://github.com/apollographql/apollo-client-devtools/tree/release-3.0

**Comment by darkbasic** [comment]
They didn't, but the current extension works well anyway. I managed to get it working on beta 43, but I'm not sure what exactly did the trick. It may be because of latest version of react-native-debugger, because of disable cache in network tab or because I enabled network debugging.

**Comment by Patricio18** [comment]
I have the same problem

**Comment by eduplus** [comment]
This is not react native debugger problem, but Apollo dev tools problem. After beta 39 dev tools are not working even in Web react and chrome.

**Comment by cnmoise** [comment]
Getting the same error message as OP, GraphiQL allows me to make queries against my data store but none of the queries being made by my app show up

Running on:
React Native Debugger app version: 0.11.3
React Native version: SDK v38
Platform: iOS
Is real device of platform: Yes
Operating System: Windows

**Comment by yodaheis** [comment]
This is fixed in apollo-dev-tools version 2.3.1 which is already upgraded in the master branch as of now but hasn't been packaged and released yet.
I checked by building and running the rndebugger app from the master branch and it's working fine

**Comment by jhen0409** [comment]
Fixed in v0.11.4 and v0.10.11. Thanks @yodaheis for confirming this!
